modules/installation-restricted-network-samples.adoc

```diff
@@ -29,6 +32,11 @@ ifdef::samplesoperatoraltreg[]
 Operator.
 endif::[]
 
+Setting the registry override field in the Sample Operator configuration file to link:registry.redhat.io[registry.redhat.io] is redundant because it is already directed to link:registry.redhat.io[registry.redhat.io] for everything but Jenkins images and imagestreams. It also breaks the installation payload for Jenkins imagestreams.
+
+The Samples Operator prevents the use of the following registries for the Jenkins imagestreams: link:docker.io[docker.io], link:registry.redhat.io[registry.redhat.io], link:registry.access.redhat.com[registry.access.redhat.com], link:quay.io[quay.io].
+====
```

Comment:
Would the registries look better in a bullet list? Or would that make the node too long?

Reply:
I was tempted to put them in a bullet list originally, so I I'll go ahead and do so. It will make it easier to read. Thanks for the suggestion!